src/main/java/com/shapesecurity/bandolier/es2017/transformations/ImportExportConnector.java

```diff
@@ -636,8 +645,11 @@ public static Script combineModules(@Nonnull BundlerOptions options, @Nonnull Mo
 					if (localSource.isJust()) {
 						objectProperties = objectProperties.cons(new DataProperty(new StaticPropertyName(propertyName), new IdentifierExpression(localExportEntry.right.entries().maybeHead().fromJust().right.right.name)));
 					} else {
-						Pair<Maybe<Module>, Pair<ExportDeclaration, Variable>> remoteSource = localExportEntry.right.entries().maybeHead().fromJust();
-						objectProperties = objectProperties.cons(new DataProperty(new StaticPropertyName(propertyName), new IdentifierExpression(exported.get(remoteSource.left.fromJust()).fromJust().get(inverseExports.get(remoteSource.left.fromJust()).fromJust().get(localExportEntry.right.entries().maybeHead().fromJust().right.right.name).fromJust()).fromJust().entries().maybeHead().fromJust().right.right.name)));
+						Maybe<Pair<Maybe<Module>, Pair<ExportDeclaration, Variable>>> maybeRemoteSource = localExportEntry.right.entries().maybeHead();
+						if (maybeRemoteSource.isJust()) {
```

Comment:
Wouldn't it be nicer to just `forEach` on `localExportEntry.right.entries().maybeHead()`?